client/statsdb.js

```diff
@@ -44,7 +44,7 @@ class StatsDB {
   // / Takes the account which has executed a transaction.
   async updateExecuted(account) {
     const found = this.stats.find({ account })[0]
-    found.executed += 1
+    found.executedTransactions.push({ timestamp: Date.now() })
```

Comment:
By changing this, you broke other clients like eas.cs-cli. Please keep both properties.

Reply:
Thought eac.js-cli is independent of eac.js-client. Will fix it.